unitTests/ui/QuerySuggestPreviewTest.ts

```diff
@@ -231,5 +287,54 @@ export function QuerySuggestPreviewTest() {
         }, test.cmp.options.executeQueryDelay);
       });
     });
+    describe('Analytics', () => {
+      function getAResult(done) {
+        const previewContainer = $$(suggestionContainer.el).find('.coveo-preview-results > .CoveoResult');
+        if (!previewContainer) {
+          done.fail('No result to click. Impossible validate the analytics');
```

Comment:
I would remove this `if` block. If there is no container, line 304 will give an error.